docs/product-administration-and-configuration-guide/src/main/asciidoc/configuring-openshift-connection-timeout-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='configuring-openshift-connection-timeout-proc']
+= Configuring OpenShift connection timeout
+
+By default, the OpenShift route is configured to time out HTTP requests that are longer than 30 seconds. This may cause session timeout issues in {CENTRAL} resulting in the following behaviors:
+
+* "Unable to complete your request. The following exception occurred: (TypeError) : Cannot read property 'indexOf' of null."
+* "Unable to complete your request. The following exception occurred: (TypeError) : b is null."
+* A blank page is displayed when clicking the *Project* or *Server* links in {CENTRAL}.
+
+All {CENTRAL} templates already include extended timeout configuration.
+
+To configure longer timeout on {CENTRAL} OpenShift routes, add the `haproxy.router.openshift.io/timeout: 60s` annotation on the target route.
+
+[source,yaml]
```

Comment:
We're not super consistent on placeholder values in code snippets anyway, but what you have here is unlike any of the variants I've seen us use so far, and actually looks exactly like what we use when we include actual doc variables that get rendered as something in the output. If you want actual place holders, what I've seen most often in our docs and what I use is either "$APPLICATION_NAME-rhdmcentr" or "<application-name>-rhdmcentr" or even "{application-name}-rhdmcentr". Again, it's all very subjective and we're all over the map anyway, but I would avoid what you have only because it looks identical to when we use actual renderable attributes.

And btw, if you're interested in renderable attributes in code snippets, you'd just add the attirubtes exactly as you have shown (assuming they're defined in the document-attributes), and then change the source definition for the snippet to "[source,yaml,subs="attributes+"]". But you probably already knew that.

Anyway, just some thoughts. Up to you in the end.